adsft/extraction.py

```diff
@@ -393,14 +391,11 @@ def open_xml(self):
             with open(self.file_input, 'rb') as fp:
                 raw_xml = fp.read()
 
-            # use soupparser to properly encode file contents
-            #  it could be utf-8, iso-8859, etc.
-            parsed_content = soupparser.fromstring(raw_xml)
-            # convert to string for ease of clean-up, convert html and LaTeX entities
-            raw_xml = tostring(parsed_content)
-            raw_xml = re.sub('(<!-- body|endbody -->)', '', raw_xml)

```

Comment:
Do you know why this would not be necessary anymore?

Reply:
Because the function we use to extract text ignores comments, so I decided we didn't need it. 

`
x = '''<html><p>some test<p> <!-- body endbody --><p>some other text</p></html>'''
from lxml.html import soupparser
y = soupparser.fromstring(x)
z = y.text_content()
z
'some test some other text'
`

Reply:
Let’s add a unit test for this if there isn’t one.

Reply:
done